src/data/markdown/translated-guides/en/03 Using k6 browser/02 Running browser tests.md

```diff
@@ -25,55 +25,69 @@ To run a simple local script:
   <CodeGroup labels={["script.js"]} lineNumbers={[true]}>
 
   ```javascript
-  import { chromium } from 'k6/experimental/browser';
+  import { browser } from 'k6/experimental/browser';
+
+  export const options = {
```

Comment:
We should explain to our users that the`executor` and setting the browser type is now mandatory.